controller-server/src/main/java/com/yahoo/vespa/hosted/controller/deployment/DeploymentTrigger.java

```diff
@@ -390,19 +366,24 @@ private boolean isRunningProductionJob(Application application) {
     }
 
     /**
-     * Returns whether the currently deployed version in the zone for the given production job is newer
-     * than the given version, in which case we should avoid an unsupported downgrade, or if it is the
-     * same version, and was successfully deployed, in which case it is unnecessary to redeploy it.
+     * Returns whether the currently deployed application in the zone for the given production job already
+     * has the given changes, in which case we should avoid an unsupported downgrade, or an unnecessary redeploy.
```

Comment:
"given changes" -> "current change"? As there's no `Change` parameter. Consider renaming to `changeAlreadyDeployed`.

Reply:
Yes, I removed that parameter. Updated doc and renamed method (to just `changeDeployed`).